### ISSUE TITLE
Update the example data in `dummy_data`

### ISSUE THREAD

**Issue by mapio** [iomega/ms2query#210]
I'm using release 1.2.4 and following the README I run

```sh
ms2query --library library_folder --download --ionmode positive 
ms2query --spectra dummy_spectra.mgf --library library_folder --ionmode positive 
```

Where `dummy_spectra.mgf` is https://github.com/iomega/ms2query/blob/main/dummy_data/dummy_spectra.mgf. 

Then I compared the output `results/dummy_spectra.csv` with `expected_results_dummy_data.csv` at https://github.com/iomega/ms2query/blob/main/dummy_data/expected_results_dummy_data.csv.

These two file have quite different contents, for instance, the list of headers is different (`feature_id`, `retention_index` and `spectrum_ids` appear only in one of the two files).

This can be confusing for a first time user. I think that updating the "dummy" test files will help to make users more confident that their installation and use is correct.

**Comment by niekdejonge** [comment]
Good point, thanks for making an issue. I will check and update accordingly

**Comment by niekdejonge** [comment]
Fixed with #215
